index.js

```diff
@@ -39,9 +39,22 @@ module.exports = {
    *  @param nemo {Object} - nemo namespace
    *  @param callback {Function} - errback function
    */
-  "setup": function (screenShotPath, nemo, callback) {
 
+   
 
+
+
+  "setup": function (config, nemo, callback) {
```

Comment:
Please see here for an example of how the `nemo` constructor handles overloading: https://github.com/paypal/nemo/blob/master/index.js#L41-L80